### DISCUSSION TITLE
How to contribute to gpt-pilot ?

### DISCUSSION THREAD

**Discussion by parjanyaacoder** [Pythagora-io/gpt-pilot#48] [General]
On creating a new branch and trying to push the branch,  it shows the following error -> 

> Permission to Pythagora-io/gpt-pilot.git denied

Can the access be made public to encourage open source contributions and improve gpt-pilot with new features and bug fixes ?

**Comment by zvone187** [comment]
Hey @parjanyaacoder , that's weird. You should be able to fork the repo and create a PR that way. Did you try forking?

**Comment by lucas-0liveira** [comment]
As of my last knowledge update in September 2021, OpenAI had not publicly released the training code for GPT-Pilot as open source. However, the company has shown interest in receiving community feedback and is continually working on refining their models. They have a track record of releasing updates and improvements over time. So, while full access to the source code may not be available at the moment, OpenAI seems committed to enhancing and perfecting GPT-Pilot. I recommend keeping an eye on OpenAI's official updates and announcements for the most recent information on this matter.
